### ISSUE TITLE
ERROR: No matching distribution found for awswrangler==2.13.0

### ISSUE THREAD

**Issue by CarlosDomingues** [awslabs/aws-data-wrangler#1137]
`pip install awswrangler==2.13.0` returns the following error: `ERROR: No matching distribution found for awswrangler==2.13.0`

`pip index versions awswrangler` returns:

```bash
awswrangler (0.3.2)
Available versions: 0.3.2, 0.3.1, 0.3.0, 0.2.6, 0.2.5, 0.2.4, 0.2.3, 0.2.2, 0.2.1, 0.2.0, 0.1.4, 0.1.3, 0.1.2, 0.1.1, 0.1.0, 0.0.25, 0.0.24, 0.0.23, 0.0.22, 0.0.21, 0.0.20, 0.0.19, 0.0.18, 0.0.17, 0.0.16, 0.0.15, 0.0.14, 0.0.13, 0.0.12, 0.0.11, 0.0.10, 0.0.9, 0.0.8, 0.0.7, 0.0.6, 0.0.5, 0.0.4, 0.0.3, 0.0.2, 0.0.1
```

It seems pip can't find the most recent versions of awswrangler. That said, [pypi correctly shows 2.13.0 as the most recent version](https://pypi.org/project/awswrangler/).

**Comment by CarlosDomingues** [comment]
`pip install awswrangler==2.13.0 -vvv` shows I'm using an unsupported Python version:

`Link requires a different Python (3.10.1 not in: '>=3.6.2,<=3.10.0'):`

Is that correct thought? Shouldn't any 3.10.x be supported?

**Comment by jaidisido** [comment]
Hi @CarlosDomingues, please refer to https://github.com/awslabs/aws-data-wrangler/issues/1126. In short, Wrangler 2.14.0, which is to be released this week, will support Python 3.10
